pinc/page_tally.inc

```diff
@@ -21,7 +21,7 @@ include_once($relPath.'User.inc');
 function get_page_tally_names()
 {
     $page_tally_names = [];
-    if (true) {
+    if (true) { /** @phpstan-ignore-line */
```

Comment:
I think this needs to get pulled out into `pgdp-production`, but I need to look at the other `R*` code.